test/test_response.py

```diff
@@ -758,19 +761,58 @@ def test_read_not_chunked_response_as_chunks(self):
         with pytest.raises(ResponseNotChunked):
             next(r)
 
-    def test_invalid_chunks(self):
+    def test_buggy_incomplete_read(self):
+        # Simulate buggy versions of Python
```

Comment:
This was fixed in Python 2.7.4 and Python 3 versions we no longer support, can we mention this here? Will be useful when we remove Python 2.7 support (or Python <2.7.9 support)

Reply:
Added a comment in the test